app/routes/index.tsx

```diff
@@ -37,6 +38,7 @@ export const routerConfig: RouteObject[] = [
     Component: AppRoute,
     children: [
       { index: true, Component: Frontpage },
+      { path: 'achievements/*', children: achievementRoute },
```

Comment:
nit: `achievementsRoute` (with an s)

Reply:
It's kind of weird that we call it "Trofeer" (trophies) on the client, but use `achievement` (prestasjoner) in the code and paths etc.